Misc/NEWS.d/next/Windows/2021-05-02-15-29-33.bpo-43984.U92jiv.rst

```diff
@@ -0,0 +1 @@
+Update winreg.SetValueEx to make sure the value set is not -1.
```

Comment:
Perhaps you could phrase this so it's easily understandable for all users. Maybe something like:
```
:meth:`winreg.SetValueEx` now leaves the target register untouched in case of conversion errors. Previously, ``-1`` would be written in case of such errors.
```

Take a look at [section 7 in the dev guide](https://devguide.python.org/documenting/#restructuredtext-primer) for more examples of how to use reST markup :)

Reply:
Sure. I'm now working on implementing all the proposed changes in this PR.

Reply:
Great! Also remember to add a test target this branch.